zerver/webhooks/github/view.py

```diff
@@ -433,6 +447,8 @@ def get_subject_based_on_type(payload: Dict[str, Any], event: str) -> str:
         )
     elif event == 'membership':
         return "{} organization".format(payload['organization']['login'])
+    elif event == 'team':
+        return "Team {}".format(payload['team']['name'])
```

Comment:
I think lower case "team" here would be more in line with how we usually do automated topics.